File: src/main/java/popa/catalin/domain/coffees/Americano.java
package popa.catalin.domain.coffees;

import popa.catalin.domain.Coffee;
import popa.catalin.domain.Ingredient;

import java.util.Arrays;
import java.util.List;

public class Americano extends Coffee {
    //americano: 4 cups water + 0.25 cups coffee-beans roasted
    public static List<Ingredient> getStandardIngredients() {
        return Arrays.asList(fourCupsWater, quarterCupBeansRoasted);
    }

    @Override
    public String drink() {
        return "Very strong";
    }
}

File: anaspoiala-932-dotnet/Lab01_CreationalPatterns/Entities/Americano.cs
using System.Collections.Generic;

namespace CoffeeMachine.Entities
{
	public class Americano : Coffee
	{
		public static new List<CoffeeIngredient> Ingredients { get; } = new List<CoffeeIngredient>
		{
			new CoffeeIngredient(4f, "cups", "water"),
			new CoffeeIngredient(0.25f, "cups", "coffee-beans", "roasted")
		};

		public override string Drink()
		{
			return $"Drinking an Americano Coffee.";
		}
	}
}
